src/modules/ekf2/ekf2_params.c

```diff
@@ -477,9 +477,10 @@ PARAM_DEFINE_INT32(EKF2_DECL_TYPE, 7);
 /**
  * Type of magnetometer fusion
  *
- * Integer controlling the type of magnetometer fusion used - magnetic heading or 3-component vector. The fuson of magnetomer data as a three component vector enables vehicle body fixed hard iron errors to be learned, but requires a stable earth field.
+ * Integer controlling the type of magnetometer fusion used - magnetic heading or 3-component vector.
+ * The fusion of magnetomer data as a three component vector enables vehicle body fixed hard iron errors to be learned, but requires a stable earth field.
```

Comment:
I modified this line to fix the typo "the fuson".

But FWIW this line is a little unhelpful in that
- it is not clear that "learning fixed hard iron errors" is a good thing
- what does it mean that your require a "stable earth field". i.e. as a user, how do I know if I have this.
- The main thing of interest here appears to be that the 3D fusion requires movement to work with any kind of reliability - so the recommend behaviour is that in "automatic" where magnetic fusion is used on the ground and 3D in the air. It might be worth saying that here too.

@bresch If I'm not wrong, want to suggest some words?